buildpipeline/DotNet-CoreFx-Trusted-Windows-NoTest.json

```diff
@@ -1,5 +1,29 @@
 {
   "build": [
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": false,
+      "displayName": "Run AgentTools/Begin.ps1",
+      "timeoutInMinutes": 0,
+      "condition": "succeededOrFailed()",
+      "task": {
+        "id": "e213ff0f-5d5c-4791-802d-52ea3e7be1f1",
+        "versionSpec": "2.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "targetType": "inline",
+        "filePath": "",
+        "arguments": "",
+        "script": "if (Test-Path \"$(AgentToolsPath)\\Begin.ps1\") {\n   \"Begin.ps1 script found. Executing...\"\n   cd $(AgentToolsPath)\n   & $(AgentToolsPath)\\Begin.ps1\n} else {\n   \"Begin.ps1 script does not exist. Moving on...\"\n}",
```

Comment:
nit: For consistency if we go with lowercase in the shell scripts I would suggest doing the same with the ps1 scripts. My preference is lowercase.

Reply:
I see you updated the name of the task but not the name of the scripts to lowercase. I would suggest doing both for consistency.

Reply:
Looks like you did do it for end.ps1 just missed the Begin.ps1 scripts in both definitions.